chia/wallet/puzzles/clawback/drivers.py

```diff
@@ -121,17 +122,15 @@ def create_merkle_solution(
 
 
 def match_clawback_puzzle(
-    uncurried: UncurriedPuzzle, inner_puzzle: Program, inner_solution: Program
+    uncurried: UncurriedPuzzle,
+    inner_puzzle: Union[Program, SerializedProgram],
+    inner_solution: Union[Program, SerializedProgram],
 ) -> Optional[ClawbackMetadata]:
     # Check if the inner puzzle is a P2 puzzle
     if MOD != uncurried.mod:
         return None
     # Fetch Remark condition
-    conditions = conditions_for_solution(
-        inner_puzzle,
-        inner_solution,
-        DEFAULT_CONSTANTS.MAX_BLOCK_COST_CLVM // 8,
```

Comment:
generally I think you should avoid making unrelated changes to PRs

Reply:
I see this as related to my change. I altered the type of `inner_puzzle` and `inner_solution` in this line, and I'd like to read it as one line.